### PR TITLE
Remove channel specifier from Firefox desktop Looker explore links (fixes #871)

### LINKED ISSUE #871: Links to Firefox desktop looker explores shouldn't include channel specifier
With @scholtzan's recent work on a Firefox desktop namespace in Looker, we now have links into the metrics explore from looker. For example:

image
https://dictionary.telemetry.mozilla.org/apps/firefox_desktop/metrics/fog_failed_idle_registration

For mobile products (like Firefox for Android), we specify a dimension corresponding to the channel's table name, since the links are specific *to* the channel:

https://github.com/mozilla/glean-dictionary/blob/1915eceab89409ebe9695b2f1fe79f59371cda56/scripts/build-glean-metadata.py#L145

However for Firefox desktop we group the links for all channels together (since they aren't seperate applications). We should just dispense with the filter for that case (people can filter on `normalized channel` after loading the explore, if they like).

I think we can fix this simply by passing `app_channel` into the aforementioned function and not adding the filter unless it's defined.

### PR DESCRIPTION
**Pull request by Iinh**
Working examples:

- https://deploy-preview-872--glean-dictionary-dev.netlify.app/apps/firefox_desktop/metrics/fog_ipc_buffer_sizes
- https://deploy-preview-872--glean-dictionary-dev.netlify.app/apps/firefox_desktop/metrics/fog_ipc_flush_durations

### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

**Comment by Iinh** [comment]
@wlach I decided to leave the [get looker link for events function](https://github.com/mozilla/glean-dictionary/blob/1915eceab89409ebe9695b2f1fe79f59371cda56/scripts/build-glean-metadata.py#L153) as is because it looks like FOG doesn't have any event metrics yet?

**Comment by wlach** [comment]
> @wlach I decided to leave the [get looker link for events function](https://github.com/mozilla/glean-dictionary/blob/1915eceab89409ebe9695b2f1fe79f59371cda56/scripts/build-glean-metadata.py#L153) as is because it looks like FOG doesn't have any event metrics yet?

Yup, that's right. For desktop we're still using legacy telemetry for events, not Glean.

We may need to rethink some of this at some point (we probably want to incorporate some notion of channel into the view for desktop in the Glean Dictionary), but this will at least get us going.

File: scripts/build-glean-metadata.py [hunk 1]
```diff
@@ -135,17 +135,18 @@ def etl_snake_case(line: str) -> str:
     return "_".join([w.lower() for w in words if w.strip()])[::-1]
 
 
-def get_looker_ping_explore_url(looker_namespaces, app_name, ping_name, table_name):
+def get_looker_ping_explore_url(
+    looker_namespaces, app_name, ping_name, table_name, app_channel=None
```
Comment (wlach) [inline scripts/build-glean-metadata.py:139]: Let's make this non-optional for now.

```suggestion
    looker_namespaces, app_name, ping_name, table_name, app_channel
```

**Review by wlach — APPROVED** [review]
Looks great! r+wc

**Pull request merged**